src/components/HighchartsChartProvider.js

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types'
+import React, { createContext, useCallback, useContext, useRef } from 'react'
+
+const throwIfUnitialized = () => {
```

Comment:
```suggestion
const throwIfUninitialized = () => {
```

or `throwIfNotInitialized` if it's more readable